paasta_tools/deployd/watchers.py

```diff
@@ -239,10 +240,10 @@ def get_service_instances_with_changed_id(marathon_client, instances, cluster):
     marathon_app_ids = list_all_marathon_app_ids(marathon_client)
     service_instances = []
     for service, instance in instances:
-        config = load_marathon_service_config(service=service,
-                                              instance=instance,
-                                              cluster=cluster,
-                                              soa_dir=DEFAULT_SOA_DIR)
+        config = load_marathon_service_config_no_cache(service=service,
```

Comment:
Because I'm in a good mood I'm going to change them. Really we should put it in a pre-commit hook though if we care.